packages/logs/src/boot/startLogs.ts

```diff
@@ -37,9 +37,8 @@ export function startLogs(initConfiguration: LogsInitConfiguration, errorLogger:
     trackNetworkError(configuration, errorObservable)
   }
 
-  const session = canUseEventBridge()
-    ? startStubLoggerSession()
-    : startLoggerSession(configuration, areCookiesAuthorized(configuration.cookieOptions))
+  const canUseCookies = areCookiesAuthorized(configuration.cookieOptions) && !canUseEventBridge()
```

Comment:
what is the relation between cookies and bridge?

Reply:
Created a startLoggerSessionStub